_docs-sources/pipelines/security/multi-account.md

```diff
@@ -0,0 +1,54 @@
+# Multiple Infrastructure-Live Repos
+
+We recommend using a single `infrastructure-live` repository for managing your organization's infrastructure.
+Sometimes, this isn't possible due to team structure, security requirements, or other limitations.
+You may choose to use multiple repos to:
+
+1. Facilitate more granular access controls
+1. Separate concerns that do not require shared configuration
+1. Ease the burden of high traffic repos (reducing the likelihood of feature branches becoming out-of-date relative to `main`)
+
+However, when using multiple repos it is more difficult to share configuration across environments so think carefully about
+your specific use case before making the decision.
+In order to accommodate multiple infrastructure repositories, Gruntwork Pipelines is configurable.
+
+## Create Additional Repos
+
+New `infrastructure-live` repositories can be created using the same process described in the
+[Hello World](../hello-world#setting-up-the-repositories) documentation.
+
+:::info
+Once the repo is created, you'll need to set up machine user access using either the existing machine user and `PIPELINES_DISPATCH` PAT token,
```

Comment:
Personally, I think we should use the word "repo" instead of "repository" throughout the docs, but either way we should be consistent, so I guess this should probably be "repository."